app/helpers/navbar_helper.rb

```diff
@@ -2,4 +2,12 @@ module NavbarHelper
   def active_class(link_path)
     current_page?(link_path) ? "active" : ""
   end
+
+  def settings_active_class(project)
```

Comment:
I don't think so, the helper intends to return an class name in case the current_page is one of 3 different pages.